pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -214,23 +233,82 @@ def default_qnode_transform(self, qnode, targs, tkwargs):
         if self.expand_transform:
             qnode.add_transform(
                 TransformContainer(
-                    self._expand_transform, targs, tkwargs, use_argnum=self._use_argnum_in_expand
+                    self._expand_transform,
+                    args=targs,
+                    kwargs=tkwargs,
+                    use_argnum=self._use_argnum_in_expand,
                 )
             )
         qnode.add_transform(
             TransformContainer(
                 self._transform,
-                targs,
-                tkwargs,
-                self._classical_cotransform,
-                self._is_informative,
-                self._final_transform,
+                args=targs,
+                kwargs=tkwargs,
+                classical_cotransform=self._classical_cotransform,
+                plxpr_transform=self._plxpr_transform,
+                is_informative=self._is_informative,
+                final_transform=self._final_transform,
             )
         )
         return qnode
 
+    def plxpr_transform(self, primitive, tracers, params, targs, tkwargs, state):
```

Comment:
When and where does state get initialized?

Reply:
In `TransformInterpreter` (see #6389). Everything related to `plxpr_transform` is set up for that PR.